src/TypeSpec.Extension/Emitter.Csharp/src/options.ts

```diff
@@ -120,6 +122,7 @@ export const NetEmitterOptionsSchema: JSONSchemaType<NetEmitterOptions> = {
 };
 
 const defaultOptions = {
+    "api-version": "latest",
```

Comment:
I didn't find the change in emitter.ts, are we able to load it from configuration?

Reply:
> I didn't find the change in emitter.ts, are we able to load it from configuration?

Yes. It's `api-version` in `tspconfig.yaml`